python/GafferImageUI/LookTransformUI.py

```diff
@@ -0,0 +1,92 @@
+##########################################################################
+#
+#  Copyright (c) 2022, Image Engine Design Inc. All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are
+#  met:
+#
+#	  * Redistributions of source code must retain the above
+#		copyright notice, this list of conditions and the following
+#		disclaimer.
+#
+#	  * Redistributions in binary form must reproduce the above
+#		copyright notice, this list of conditions and the following
+#		disclaimer in the documentation and/or other materials provided with
+#		the distribution.
+#
+#	  * Neither the name of John Haddon nor the names of
+#		any other contributors to this software may be used to endorse or
+#		promote products derived from this software without specific prior
+#		written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+import Gaffer
+import GafferImage
+import IECore
+
+Gaffer.Metadata.registerNode(
+
+	GafferImage.LookTransform,
+
+	"description",
+	"""
+	OpenColorIO LookTransform\n"
+	A 'look' is a named color transform, intended to modify the look of an
+	image in a 'creative' manner (as opposed to a colorspace definion which
+	tends to be technically/mathematically defined).\n
+	Examples of looks may be a neutral grade, to be applied to film scans
+	prior to VFX work, or a per-shot DI grade decided on by the director,
+	to be applied just before the viewing transform.\n
+	OCIOLooks must be predefined in the OpenColorIO configuration before usage,
+	often reference per-shot/sequence LUTs/CCs and are applied in scene linear colorspace.\n
+	See the look plug for further syntax details.\n
+	See opencolorio.org for look configuration customization examples.
+	""",
```

Comment:
I'm not sure exactly why, but this is formatting very strangely in the docs and in the tooltip.

![image](https://user-images.githubusercontent.com/1133871/190413112-766486f0-74e1-4b12-9418-715219a5ab20.png)

We can definitely omit the `OpenColorIO LookTransform\n"` heading, since everywhere we show the description the heading is already implied (and the stray `"` there is definitely part of the weird formatting I think). And I think maybe it's also getting confused by the `\n` since we're already inside a triple-quoted string and can just use regular newlines. If I tidy those both up, the formatting seems OK.

Reply:
yeah, that was ugly. managed to get the formatting look much nicer now :)